.changeset/moody-jobs-beg.md

```diff
@@ -0,0 +1,6 @@
+---
+'angular-example': patch
+'amplify-ui-angular-mono': patch
```

Comment:
We don't release either of these packages, so this will break changesets if it's included. we may want to include `@aws-amplify/ui-angular` if we really decide to bump the devkit version in packages/angular/package.json

Reply:
Done.